azurerm/resource_arm_app_service.go

```diff
@@ -656,12 +668,28 @@ func flattenAppServiceAppSettings(input map[string]*string) map[string]string {
 }
 
 func expandAzureRmAppServiceIdentity(d *schema.ResourceData) *web.ManagedServiceIdentity {
-	identities := d.Get("identity").([]interface{})
+	v := d.Get("identity")
+	identities := v.([]interface{})
```

Comment:
what is the reasoning behind splitting this onto to lines?
```suggestion
	identities := d.Get("identity").([]interface{})
```

Reply:
No reasoning, it came from copy-paste.